src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml

```diff
@@ -224,6 +225,7 @@
         <Style x:Name="KGF_TextBlockStyle" TargetType="TextBlock">
             <Setter Property="FontWeight" Value="Normal"/>
             <Setter Property="FontSize" Value="14"/>
+            <Setter Property="IsTextSelectionEnabled" Value="True"/>
```

Comment:
NIT: The text `Analysis is not supported for this function.` can now be selected. We should probably not use this style for this text, the context is very different.